Type, Accessibility fix, Typos

ramsbaron opened the pull request:
This PR includes three changes

1. Fix for the accessibility check.
2. Added a new property stop_on_a11y_fail to explicitly stop the scenario if accessibility threshold defined by MIN_ACCESSIBILITY_SCORE is not met,
3. Typos

mkmurali commented on lib/ally_steps.js:
```diff
@@ -0,0 +1,18 @@
+const assert = require("assert");
+const MIN_ACCESSIBILITY_SCORE = process.env.min_accessibility_score || 80;
+
+step("Run a11y check", async () => {
```
This will break existing test cases. Please add "Run a11y check" as an alias, to be backward compatible.

mkmurali commented on lib/ally_steps.js:
```diff
@@ -0,0 +1,18 @@
+      5
+    )}`
+  );
+  if (process.env.stop_on_a11y_fail === "true") {
```
For backward compatibility, set the default value of this property to fail. This will otherwise break existing behavior.  Also update this in README to highlight the option.

ramsbaron replied:
Added an alias in a more user friendly language

mkmurali reviewed (changes requested):
Please see notes for a couple of changes requested.

ramsbaron commented:
1. Added alias for the a11y check step
2. Set the env var to continue scenario if a11y check fails due to min score not met
2. Formatted the a11y check output to show better in gauge report
3. Added documentation for a11y check and related properties

mkmurali reviewed (approved):
lgtm
